### PR TITLE
feat: introduce StatusWalletColorSelect component

### LINKED ISSUE #1497: Wallet: account color picker does not match figma design
image

### PR DESCRIPTION
**Pull request by PascalPrecht**
This is a new form control component to select colors for wallet
accounts. It will replace the currently used `ColorSelect` in the
Status Desktop application so it aligns with the new designs.

Closes #1497

### CODE REVIEW

**Comment by PascalPrecht** [comment]
Here's what it looks like in light and dark themes:
Screenshot 2021-01-20 at 17 27 23
Screenshot 2021-01-20 at 17 27 00

File: ui/shared/status/StatusWalletColorSelect.qml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        anchors.top: parent.top
+        font.pixelSize: 13
+        height: 18
+    }
+
+    RowLayout {
+        id: colors
+        spacing: 6
+        anchors.top: label.bottom
+        anchors.topMargin: 8
```
Comment (jrainville) [inline ui/shared/status/StatusWalletColorSelect.qml:29]:
```suggestion
        anchors.topMargin: Style.current.halfPadding
```

File: ui/shared/status/StatusWalletColorSelect.qml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import "../../imports"
+import "../../shared"
+import "../../shared/status"
+
+Item {
+    id: control
+    property string selectedColor
+    property string label: qsTr("Account color")
+    property var model
+    height: label.height + colors.height + 8
```
Comment (jrainville) [inline ui/shared/status/StatusWalletColorSelect.qml:13]: `childrenRect.height` should work since you don't use `anchors.fill` or `veriticalCenter`, etc.

**Review by jrainville — APPROVED** [review]
Approving since my comments are just good to have

**Pull request merged**